frontend/packages/core/src/chip.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { ChipProps as MuiChipProps } from "@material-ui/core";
+import { Chip as MuiChip } from "@material-ui/core";
+
+type variants = "error" | "unknown" | "pending" | "running" | "succeeded";
+export interface ChipProps extends Pick<MuiChipProps, "label"> {
+  variant: variants;
```

Comment:
nit: you can inline the variants here since they aren't being used anywhere else.